src/createUseMemo.lua

```diff
@@ -19,11 +19,11 @@ local function createUseMemo(useValue)
 		if needToRecalculate then
 			currentValue.value = {
 				dependencies = dependencies,
-				memoizedValue = createValue(),
+				memoizedValue = {createValue()},
 			}
 		end
 
-		return currentValue.value.memoizedValue
+		return unpack(currentValue.value.memoizedValue) or nil
```

Comment:
Why `or nil`?

This is going to cause functions like `function x() end` to return nil instead of void, which is a potentially surprising behavior.